schedule/yast/remote_vnc_controller_opensuse.yaml

```diff
@@ -2,6 +2,8 @@
 name:           remote_vnc_controller
 description:    >
   Install remote server (parallel job) with vnc.
+vars:
+  QEMUVGA: cirrus
```

Comment:
This should be defined in MACHINE. Please schedule on `64bit_cirrus` machine, if needed.

Reply:
changed applied df056d4b8